src/shipit_static_analysis/shipit_static_analysis/report/mozreview.py

```diff
@@ -110,6 +113,7 @@ def mozreview_publish(issue):
 
         # Publish the review
         # without ship_it to avoid automatically r+
+        stats.api.increment('report.mozreview')
```

Comment:
You want to count only the cases where the review is published on mozreview, correct?

Reply:
(That is, if there are no issues to publish, you don't want to increment this number)

Reply:
Yes, only when there are some issues to publish (this line is never reached when there are no publishable issues, see the else case just before)

Reply:
Cool, just wanted to confirm